modules/fundamental/src/purl/model/details/purl.rs

```diff
@@ -312,6 +313,7 @@ pub struct PurlStatus {
     pub vulnerability: VulnerabilityHead,
     pub average_severity: Severity,
     pub average_score: f64,
+    pub scores: Vec<Score>,
```

Comment:
I know we're doing a bad job with this so far. But it would be good to improve in the future and document fields, so that it would also end up in the OpenAPI spec

Reply:
Updated the description. Made me notice that it feels a bit odd for the scores/severity to be in the PurlStatus type given that theres a 1:N from vuln->score, but no direct relationship between purl_status->score from what I saw in the seaorm schema